### DISCUSSION TITLE
Can't use MediaPlayerElement, but official documents seems available. Help please

### DISCUSSION THREAD

**Discussion by joyful-jisu** [unoplatform/uno#10852] [Q&A] [answered]
Hello. guys. Thank you for developing wonderful cross-platform framework.

## Developing what
I am developing an image and video editing tool that uses machine learning algorithms at my company.
I'm trying to apply an algorithm for each frame, draw a stroke like inkcanvas on it, or partially apply a blur effect.

## Problem
I am trying to implement this through MediaPlayerElement or Canvas, but MediaPlayerElement is currently unable to build in my development environment.I would appreciate it if you could help me to use MediaPlayerElement.

(I saw an issue, pull-request and, UITest code, official reference on Unoplatform site about `MediaPlayerElement`. So i thought this component was implemented.) 

### Not showing up on intellisense
![image](https://user-images.githubusercontent.com/97220173/209945400-b79102ca-e5fc-47fe-b4d3-61a3cb17ed94.png)
### Can't build
> Build failed all platform(Windows, WASM, Mobile, Linux)

![image](https://user-images.githubusercontent.com/97220173/209945506-154423b3-4c63-42a3-b1f6-2e248d1ddf74.png)

## Need advice
In addition, it would be nice if you could give us some advice on which UI component implemented in Uno should be used to implement the above-mentioned frame-by-frame editing and the function of giving partial effects with the mouse.

## My develop environment
* OS : Windows 11
* IDE : Visual Studio 2022 Community preview 17.5
* Uno-Check : 1.10.0.0 (All check passed)
* .NET : 7.0
* Platform :  All checked (Mobile: android, ios, WASM, linux.skia, WPF, Windows)
* Uno template - Uno platform App (WinUI)
![image](https://user-images.githubusercontent.com/97220173/209940256-d753d521-16c8-44be-8d7c-53b7e020ed94.png)

### Package Reference on <Projects>.Windows.csproj
![image](https://user-images.githubusercontent.com/97220173/209945695-99541e2b-ddb8-4d95-ab51-b9cc74d06ae4.png)

**Comment by jeromelaban** [comment, marked as answer]
The `MediaPlayerElement` has been added in [WinAppSDK 1.2](https://blogs.windows.com/windowsdeveloper/2022/11/16/whats-new-in-windows-app-sdk-1-2/), you'll need to upgrade to be able to use it.

**Reply by joyful-jisu** [reply]
@jeromelaban  Thank you for reply!
Actually, I already tried upgrade window app sdk version before i upload the Q&A.
I checked window app sdk path, version, and package reference attribute value changed.
But after upgrade window app sdk, I can't build anything because of error(Sorry, I can't remember full error log. it was something like 'targeting sdk is not compatible'.
I found answer you wrote on uno repository issue. You suggest change FrameworkReference key's RuntimeFrameworkVersion and TargetingPackVersion value to 10.0.22000.25 -> 10.0.22000.27. 
I followed your guide and it worked! Thanks!
Since I was new to Windows application development, I didn't know I needed to modify those variables separately because I installed them according to the guide and upgraded them through Visual Studio's Nuget installer.
